src/system/ui/pages/post/index.js

```diff
@@ -204,6 +204,15 @@ export const Post = ( { location, match, history } ) => {
 							value: item.date
 
 						},
+						post_author: {
+							label: __( 'Author' ),
+							labelPlacement: 'beside',
+							component: 'select',
+							id: 'post_author',
+							options: item.authorList,
+							value: item.post_author,
```

Comment:
Passing a value in the field config doesn't actually do anything. The form hook will pick that up automatically in the `defaults` object that is passed. You can override it there too.

Reply:
Removed the value field and hookup the item with the default object.